app/views/unattended/provisioning_templates/provision/windows_default_provisioning.erb

```diff
@@ -70,9 +74,23 @@ description: |
         <component name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
             <UserAccounts>
                 <AdministratorPassword>
-                    <PlainText>false</PlainText>
                     <Value><%= root_pass %></Value>
+                    <PlainText>false</PlainText>
                 </AdministratorPassword>
+            <% if host_param('ansible_user') and host_param('create_ansible_user') -%>
+                <LocalAccounts>
+                    <LocalAccount wcm:action="add">
+                        <Password>
+                            <Value><%= host_param('ansible_ssh_pass') %></Value>
```

Comment:
I wonder if anyone has `<` in their password and you need to XML escape values

Reply:
Yeah, you are right, we need to fix this as well.

Reply:
Added HTML escaping for Windows https://www.w3.org/TR/REC-xml/#sec-cdata-sect